legal-api/src/legal_api/models/business.py

```diff
@@ -335,7 +335,7 @@ def good_standing(self):
         last_ar_date = self.last_ar_date or self.founding_date
         is_active = self.state.name == Business.State.ACTIVE.name
         # Good standing is if last AR was filed within the past 1 year, 2 months and 1 day and is in an active state
-        return last_ar_date + datedelta.datedelta(years=1, months=2, days=1) > datetime.utcnow() and is_active
+        return last_ar_date + datedelta.datedelta(years=1, months=2, days=1) > datetime.utcnow() if is_active else True
```

Comment:
It looks like this fixes the issue for [14733](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/14733)